metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/ExecutionProgress.java

```diff
@@ -91,6 +91,6 @@ public void setStatus(TaskState status) {
 
   public void recalculateProgressPercentage() {
     this.progressPercentage = this.expectedRecords == 0 ? 0
-        : Math.round(100f * this.processedRecords / this.expectedRecords);
+        : Math.round(100f * (this.processedRecords + this.deletedRecords)/ (this.expectedRecords + this.deletedRecords));
```

Comment:
This could cause confusion if there are large numbers of deleted records, right? The user could see something like 1 record done out of 100, and the bar is at 95%.

Reply:
I see the point.
The way I thought of this is that if there are many deleted records being "processed" the task will seem stuck in the progress bar until those deleted go through.
I'm fine with either, it's more a matter of display, let me know what you think.

Reply:
Ok, maybe we should leave it like this. I think we may wish to eventually get the expected deleted records count and solve both the problems/comments at once.